Scripts/cours.js

```diff
@@ -65,38 +65,43 @@ function deleteClass(id){
  if(idClass !== -1){
     classes.splice(idClass, 1);
     deleteAll();
-    getClasses();
+    displayClasses();
  }else{
     alert("Not found");
  }
 }
 
-function postClass(){
+function createClass(){
     const title = document.getElementById("titleInput").value;
     const period = document.getElementById("periodInput").value;
     const nbHour = document.getElementById("nbHourInput").value;
     const teacher = document.getElementById("teacherInput").value;
     const description = document.getElementById("descriptionInput").value;
 
-    if(!title || !period || !teacher || !description || !Number.isInteger(+nbHour)){
+    // Ctrl input
+    // Number.isInteger(+nbHour) => le + permet de retirer les ""
+    if(!title || !period || !teacher || !description || !nbHour)
+    {
         alert("Vous n'avez pas remplis tous les champs");
-        return;
+        return; 
     }
+    const hourOnly = nbHour.substring(0, nbHour.length -1);
```

Comment:
you should check if hourOnly is an integer

Reply:
Done!